lib/rails_multitenant/global_context_registry.rb

```diff
@@ -171,7 +173,7 @@ def get(symbol)
     # Duplicate the registry
     def duplicate_registry
       globals.each_with_object({}) do |(key, value), result|
-        result[key] = (value.nil? || value.is_a?(Fixnum)) ? value : value.dup
+        result[key] = (value.nil? || value.is_a?(INTEGER_CLASS)) ? value : value.dup
```

Comment:
Doesn't `Fixnum` subclass `Integer` in Ruby pre-2.4? If so, can we just convert this to `value.is_a?(Integer)`?

Reply:
Good point. Changed.